### PR TITLE
Ensure WebAnimation polyfill is installed for amp-story

### PR DESCRIPTION
**Pull request by alanorozco**
Fixes animations not running on Safari.

### CODE REVIEW

File: extensions/amp-animation/0.1/web-animations-polyfill.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import {installWebAnimations} from 'web-animations-js/web-animations.install';
+
+
+const POLYFILLED = '__AMP_WA';
+
+
+/**
+ * @param {!Window} win
+ */
+export function ensurePolyfillInstalled(win) {
```
Comment (choumx) [inline extensions/amp-animation/0.1/web-animations-polyfill.js]: Nit: `installWebAnimationsIfNecessary`

Comment (alanorozco) [reply extensions/amp-animation/0.1/web-animations-polyfill.js]: Done.

File: extensions/amp-animation/0.1/web-animations-polyfill.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {installWebAnimations} from 'web-animations-js/web-animations.install';
+
+
+const POLYFILLED = '__AMP_WA';
```
Comment (choumx) [inline extensions/amp-animation/0.1/web-animations-polyfill.js]: Consider a more specific identifier, e.g. `__AMP_WA_POLYFILL`.

Comment (alanorozco) [reply extensions/amp-animation/0.1/web-animations-polyfill.js]: Noted. I think I'd rather leave the constant that was in place.

**Pull request merged**